test/tests_for_sdkmr/ee2_load_test.py

```diff
@@ -245,13 +245,18 @@ def update_states(index, job_ids_queued, job_ids_running, job_ids_finish):
 
     @patch.object(Condor, "run_job", return_value=si)
     @patch.object(WorkspaceAuth, "can_write", return_value=True)
+    @patch(
+        "installed_clients.CatalogClient.Catalog.list_client_group_configs",
+        autospec=True,
+    )
     @patch("installed_clients.CatalogClient.Catalog.get_module_version", autospec=True)
     @patch("installed_clients.CatalogClient.Catalog.log_exec_stats", autospec=True)
-    def test_run_job_stress(self, ccles, cc, workspace, condor):
+    def test_run_job_stress(self, ccles, ccgmv, cclcgc, workspace, condor):
         """
         testing running 3 different jobs in multiple theads.
         """
-        cc.return_value = {"git_commit_hash": "moduleversiongoeshere"}
+        ccgmv.return_value = {"git_commit_hash": "moduleversiongoeshere"}
+        cclcgc.return_value = []
```

Comment:
can we rename these clients? It's a bit confusing

Reply:
done